x-pack/plugins/security_solution/public/resolver/view/panels/related_event_detail.tsx

```diff
@@ -62,6 +62,7 @@ const objectToDescriptionListEntries = function* (
 const StyledDescriptionList = memo(styled(EuiDescriptionList)`
   &.euiDescriptionList.euiDescriptionList--column dt.euiDescriptionList__title.desc-title {
     max-width: 8em;
+    word-break: break-all;
   }
```

Comment:
:information_source: Alternatively, we could do something along these lines if we feel it's warranted:
https://github.com/elastic/security-team/issues/21#issuecomment-668609940

Reply:
@monina-n indicated she wants this, so I'm going to adjust to this approach. https://github.com/elastic/security-team/issues/70#issuecomment-680261359

Reply:
why not the formatted text approach? it's best of both worlds imo

Reply:
@kqualters-elastic you mean GeneratedText from the example there or something else?

Reply:
Not sure if that's what you meant, but Robert asked for it so I put it in here: https://github.com/elastic/kibana/pull/75926/commits/b64164f7f5e244ecad6e0e8c46831fe65e564a9b